astropy/modeling/tests/test_models.py

```diff
@@ -327,9 +332,13 @@ def test_deriv_2D(self, model_class, test_parameters):
         fitter_no_deriv = fitting.LevMarLSQFitter()
         new_model_no_deriv = fitter_no_deriv(model_no_deriv, xv, yv, data,
                                              estimate_jacobian=True)
-        assert_allclose(new_model_with_deriv.parameters,
-                        new_model_no_deriv.parameters,
-                        rtol=0.1)
+        assert_allclose(new_model_with_deriv(xv_test, yv_test),
+                        new_model_no_deriv(xv_test, yv_test),
+                        rtol=1e-3)
```

Comment:
```suggestion
                        rtol=1e-2)
```
This may fix the current failures in this test in some of the CI.